packages/google_atlas/lib/src/google_atlas.dart

```diff
@@ -105,6 +122,28 @@ class _GoogleMapsProviderState extends State<GoogleMapsProvider> {
     return googleMarkers;
   }
 
+  /// Converts an `Atlas.Circle` to a `GoogleMaps.Circle`
+  Future<Set<GoogleMaps.Circle>> _toGoogleCircles(Set<Circle> circles) async {
+    Set<GoogleMaps.Circle> googleCircles = Set();
```

Comment:
I think this would read a bit better if you use a `map` function, rather than a `for-in` loop